web/html/src/utils/test-utils/screen.ts

```diff
@@ -16,6 +16,28 @@ type GenericScreen = {
         : Screen[Key];
 };
 
+/**
+ * Labels (as in `components/input/Label.tsx`) often have a required suffix "*" and a generic display suffix ":".
+ * Instead of cluttering the DOM there, we account for it in the tests by using a normalizer here.
+ * See: https://testing-library.com/docs/queries/about/#normalization
+ */
+const defaultNormalizer = getDefaultNormalizer();
+const labelNormalizer = (input: string) => {
+    return defaultNormalizer(input)
+    .replace(/:$/, "") // Remove trailing ":"
+    .replace(/ \*$/, ""); // Remove trailing " *"
+};
+
+const getByLabelText = rawScreen.getByLabelText;
+type GetByLabelTextArgs = Parameters<typeof rawScreen.getByLabelText>;
+Object.assign(rawScreen, {
```

Comment:
Can we just hint developers we are overriding/rewriting the `getByLabelText` method with a simple comment:
```// Override "getByLabelText"```

It's a typical thing in many languages, I'd like to have it here as well, if you don't mind.